doc/api/events.md

```diff
@@ -155,9 +155,60 @@ myEmitter.emit('error', new Error('whoops!'));
 // Prints: whoops! there was an error
 ```
 
+## AutoCatch of Promise rejection
+
+Using async functions in event handlers is problematic, because it would
+lead to `'unhandledRejection'` in case of an exception
+
+```js
+const ee = new EventEmitter();
+ee.on('something', async (value) => {
+  throw new Error('kaboom');
+});
+```
+
+The `autoCatch` option in the `EventEmitter` constructor or the global
+setting change this behavior, installing a `.catch()` handler on the
+`Promise`. This `catch` handler routes the exception to the
+[`'rejection'`][rejection] event if there is a listener, or to
```

Comment:
This makes the `'rejection'` event special no? Someone might use this event for something else. Perhaps it's better to use a symbol?